src/main/resources/podcast.xml.mustache

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
neato

fun fact: we used Mustache (+ React.js) in Yandex.Mail Android app liyek 3 years ago, was fun